README.md

```diff
@@ -328,6 +329,32 @@ A value which has a Group must provide an `invert` method. The
 
 1. `invert` must return a value of the same Group.
 
+### Filterable
+
+1. `v.filter(x => p(x) && q(x))` is equivalent to `v.filter(p).filter(q)` (distributivity)
```

Comment:
This law actually `Functor` laws, if you replace `a -> Boolean' to monoid `Conjunction`.